src/components/StylesFromLiteralsService.js

```diff
@@ -120,7 +119,8 @@ goog.provide('ga_stylesfromliterals_service');
           minResolution: getMinResolution(value),
           maxResolution: getMaxResolution(value),
           labelProperty: getLabelProperty(value.vectorOptions.label),
-          labelTemplate: getLabelTemplate(value.vectorOptions.label)
+          labelTemplate: getLabelTemplate(value.vectorOptions.label),
+          imageRotation: value.rotation
```

Comment:
`imageRotationProperty` is more understandable.

Reply:
I agree.